tests/integration/awslambda/test_lambda_api.py

```diff
@@ -45,8 +47,127 @@ def fixture_snapshot(snapshot):
     )
 
 
+def string_length_bytes(s: str) -> int:
+    return len(s.encode("utf-8"))
+
+
+def environment_length_bytes(e: dict) -> int:
+    serialized_environment = json.dumps(e, separators=(":", ","))
+    return string_length_bytes(serialized_environment)
+
+
 @pytest.mark.skipif(is_old_provider(), reason="focusing on new provider")
 class TestLambdaFunction:
+    @pytest.mark.aws_validated
```

Comment:
One, last nitpick: TestLambdaFunction is currently skipped on the old provider. Can you move it to your own class (like `TestLambdaSizeLimits`) for now?

Reply:
Done 👍